src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp

```diff
@@ -824,48 +835,50 @@ ov::element::Type eltwise_precision_helper::get_precision(const size_t inputs_nu
-        OV_CASE(Algorithm::EltwiseLogicalOr, jit_logical_or_emitter),
-        OV_CASE(Algorithm::EltwiseLogicalNot, jit_logical_not_emitter),
-        OV_CASE(Algorithm::EltwiseLogicalXor, jit_logical_xor_emitter),
-        OV_CASE(Algorithm::EltwiseMaximum, jit_maximum_emitter),
-        OV_CASE(Algorithm::EltwiseMinimum, jit_minimum_emitter),
-        OV_CASE(Algorithm::EltwiseMish, jit_mish_emitter),
-        OV_CASE(Algorithm::EltwiseMod, jit_mod_emitter),
-        OV_CASE(Algorithm::EltwiseMulAdd, jit_mul_add_emitter),
-        OV_CASE(Algorithm::EltwiseMultiply, jit_multiply_emitter),
-        OV_CASE(Algorithm::EltwisePrelu, jit_prelu_emitter),
-        OV_CASE(Algorithm::EltwisePowerStatic, jit_power_static_emitter),
-        OV_CASE(Algorithm::EltwiseRoundHalfAwayFromZero, jit_round_half_away_from_zero_emitter),
-        OV_CASE(Algorithm::EltwiseRoundHalfToEven, jit_round_half_to_even_emitter),
-        OV_CASE(Algorithm::EltwiseSelect, jit_select_emitter),
-        OV_CASE(Algorithm::EltwiseSigmoid, jit_sigmoid_emitter),
-        OV_CASE(Algorithm::EltwiseSoftSign, jit_soft_sign_emitter),
-        OV_CASE(Algorithm::EltwiseSqrt, jit_sqrt_emitter),
-        OV_CASE(Algorithm::EltwiseSubtract, jit_subtract_emitter),
-        OV_CASE(Algorithm::EltwiseSwish, jit_swish_emitter),
-        OV_CASE(Algorithm::EltwiseTanh, jit_tanh_emitter));
+    OV_SWITCH(intel_cpu,
+              SupportedPrecisions,
+              precisions,
+              algo,
+              OV_CASE(Algorithm::EltwiseRelu, jit_relu_emitter),
+              OV_CASE(Algorithm::EltwiseAbs, jit_abs_emitter),
+              OV_CASE(Algorithm::EltwiseAdd, jit_add_emitter),
+              OV_CASE(Algorithm::EltwiseClamp, jit_clamp_emitter),
+              OV_CASE(Algorithm::EltwiseDivide, jit_divide_emitter),
+              OV_CASE(Algorithm::EltwiseElu, jit_elu_emitter),
+              OV_CASE(Algorithm::EltwiseEqual, jit_equal_emitter),
+              OV_CASE(Algorithm::EltwiseExp, jit_exp_emitter),
+              OV_CASE(Algorithm::EltwiseFloor, jit_floor_emitter),
+              OV_CASE(Algorithm::EltwiseFloorMod, jit_floor_mod_emitter),
+              OV_CASE(Algorithm::EltwiseCeiling, jit_ceiling_emitter),
+              OV_CASE(Algorithm::EltwiseGeluErf, jit_gelu_erf_emitter),
+              OV_CASE(Algorithm::EltwiseGeluTanh, jit_gelu_tanh_emitter),
+              OV_CASE(Algorithm::EltwiseGreater, jit_greater_emitter),
+              OV_CASE(Algorithm::EltwiseGreaterEqual, jit_greater_equal_emitter),
+              OV_CASE(Algorithm::EltwiseHswish, jit_hswish_emitter),
+              OV_CASE(Algorithm::EltwiseIsFinite, jit_is_finite_emitter),
+              OV_CASE(Algorithm::EltwiseIsInf, jit_is_inf_emitter),
+              OV_CASE(Algorithm::EltwiseIsNaN, jit_is_nan_emitter),
+              OV_CASE(Algorithm::EltwiseLessEqual, jit_less_equal_emitter),
+              OV_CASE(Algorithm::EltwiseLogicalAnd, jit_logical_and_emitter),
```

Comment:
For some reason EltwiseLogicalOr is deleted from this list. It leads to failed tests. Please return it back.

Reply:
Not sure what caused this. It's not the style check because after adding it back and running the style check, it didn't get deleted or anything. However I did notice it disappear momentarily and reappear when I was rebasing my branch after syncing master. Anyway, it should be there this time.

Reply:
By the way, this indeed is happening due to `clang-format`. When my colleague built https://github.com/openvinotoolkit/openvino/pull/27841 with `clang_format_fix_all`, `EltwiseOr` disappeared from both places. Not sure about the reason for this behavior though.